cmd/clusterctl/phases/applyaddons.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/clusterdeployer/clusterclient"
+)
+
+func ApplyAddons(client clusterclient.Client, addons string) error {
+	glog.Info("Applying Addons")
+	if err := client.Apply(addons); err != nil {
+		return fmt.Errorf("unable to apply addons: %v", err)
```

Comment:
Maybe this should just return err since clusterctl as written will print "unable to apply addons to target cluster: unable to apply addons: %v" which is redundant. It would basically move this string up into RunAlphaPhaseApplyAddons. WDYT?